packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -1225,6 +1225,8 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			if ((event.metaKey || event.ctrlKey) && event.shiftKey && event.code === 'KeyV') {
 				event.preventDefault();
 				pasteAsPlainText(null);
+			} else if ((event.metaKey || event.ctrlKey) && event.altKey && event.key === '-') {
+				editor.execCommand('InsertUnorderedList');
```

Comment:
Be aware that users can customize the shortcut for the "insert unordered list" command in settings.

Note that we currently need to register Joplin's keyboard shortcuts with the markdown editors (see [useKeymap.ts](https://github.com/laurent22/joplin/blob/c10e617870523e4734aee36ac90e4240496a0eaf/packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/v5/utils/useKeymap.ts#L56)). It might be necessary to do something similar with TinyMCE.   

Thank you for working on this!

Reply:
Right, I guess that fixes some issue but it doesn't seem proper to hard-code shortcuts. Isn't it possible to either define a command for it and get any existing command working in this context?

Reply:
@RafaR13, any thoughts on this? I'm not sure we should merge this pull request in its current state

Reply:
I'm sorry for the delayed response, @laurent22.
As personalizedrefrigerator said, it might be necessary to do something similar to what is being done in markdown editors, but in TinyMCE. However, I don't quite understand why every other editor command that I've tried works but the bulleted list doesn't. This makes me believe that the commands are indeed being treated somewhere, which wouldn't require an implementation similar to the one in the markdown editors. Nonetheless, I've investigated the code and found no particular reason that would justify the fact that this specific command isn't being treated properly.

Additionally, in response to your comment regarding hardcoding shortcuts, I agree with you that it is not the best practice. However, something similar is being implemented inside TinyMce.tsx, where three shortcuts are being hardcoded to create the three types of lists (Ctrl+Shift+7, Ctrl+Shift+8, and Ctrl+Shift+9 respectively).

Despite what I've stated, I respect whatever decision you make regarding this pull request.

Thank you for your time.

Reply:
> However, I don't quite understand why every other editor command that I've tried works but the bulleted list doesn't.

I've also noticed that `textBulletedList` can't be run from <kbd>ctrl</kbd>-<kbd>p</kbd> in the Rich Text editor, but can from the markdown editor (testing in Joplin 2.14).

Reply:
Looking into this further, I think that `textBulletedList` will likely need to be mapped to a TinyMCE command in this file:

https://github.com/laurent22/joplin/blob/dev/packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/utils/joplinCommandToTinyMceCommands.ts